src/desktop/DesktopMain.js

```diff
@@ -82,6 +82,17 @@ if (process.argv.indexOf("-r") !== -1) {
 				wm.getAll().forEach(w => w.show())
 			}
 			handleArgv(args)
+		}).on('open-url', (e, url) => {
+			// MacOS mailto handling
+			e.preventDefault()
+			if (!url.startsWith('mailto:')) {
```

Comment:
It looks like all other URLs should not work , maybe we should comment if they still do.